data/newMigrations/001-programs.js

```diff
@@ -0,0 +1,16 @@
+exports.up = function (knex) {
+    return knex.schema.createTable('programs', tbl => {
+        tbl.increments('program_id')
+            .primary();
```

Comment:
When doing `.increments` knex actually automatically makes it the primary key behind the scenes, so you don't need `.primary()`.